tests/distributions/test_beta_binomial.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import torch
+
+from scvi.distributions import BetaBinomial
+
+
+def test_beta_binomial_distribution():
+    alpha = torch.ones(size=(2,))
+    beta = torch.ones_like(alpha)
+
+    num_trials = 100
+    total_count = torch.ones_like(alpha) * num_trials
+    x = torch.randint_like(total_count, high=num_trials)
+
+    dist_param_1 = BetaBinomial(total_count=total_count, alpha=alpha, beta=beta)
+
+    log_p_alpha_beta = dist_param_1.log_prob(x)
+
+    mu = alpha / (alpha + beta)
+    gamma = 1 / (alpha + beta + 1)
+
+    dist_param_2 = BetaBinomial(total_count=total_count, mu=mu, gamma=gamma)
+
+    log_p_mu_gamma = dist_param_2.log_prob(x)
+    assert (log_p_alpha_beta - log_p_mu_gamma).abs().max().item() <= 1e-8
+
+    # Should fail as no parameterization is specified
+    with pytest.raises(ValueError):
```

Comment:
also need to test for `ValueError` when both parameterizations are provided as well as overlaps (e.g. `total_counts`, `beta`, and `gamma` are provided).

Reply:
Added a corresponding test